README.md

```diff
@@ -1 +1,3 @@
-# Medical_Helper
\ No newline at end of file
+# Medical Help - No excuses for delays in taking medicines
```

Comment:
Good to go as of now, but I would recommend you to make it more detailed in future.